packages/yoga/src/Divider/native/Divider.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import theme from '../../Theme/helpers/themeReader';
+
+const StyledDivider = styled.View`
+  ${props => {
+    const { borders, colors } = theme(props);
+
+    return css`
+      height: ${borders.small};
+      width: 100%;
+      background-color: ${colors.light};
+    `;
+  }}
+`;
+
+const DividerContainer = styled.View`
+  flex-direction: row;
+`;
+
+const Divider = props => {
+  return (
+    <DividerContainer>
+      <StyledDivider {...props} />
```

Comment:
The fact that we are passing the `props` forward made me remember about the `yoga-system`, we do not need any `system` implementation at the `Divider`?